openshift_scalability/content/statefulset-pv-template.json

```diff
@@ -51,6 +51,16 @@
               {
                 "name": "server",
                 "image": "${IMAGE}",
+                "resources": {
+                  "requests": {
+                    "memory": "128Mi",
```

Comment:
what is the flow, in terms of configuration to run, there is some args like ${image} which injected from somewhere, we can inject these limits as well ?